misc/ci_setup.py

```diff
@@ -160,7 +160,7 @@ def run(self):
 
         check_command_existence("wget")
         try:
-            import pip
+            import pip  # noqa
```

Comment:
```suggestion
            import pip  # noqa: F401
```

You can use this to specify the rule that's ignored. It may be useful to still apply the other rules.

Reply:
I would definitely recommend doing this